stylesheets/_session.scss

```diff
@@ -983,60 +965,10 @@ label {
   border-radius: 3px;
 }
 
-.seed-dialog {
-  &__description {
-    max-width: $session-modal-size-lg;
-  }
-}
-
 .messages li {
   transition: $session-transition-duration !important;
 }
 
-.session-radio-group fieldset {
-  border: none;
-  margin-inline-start: $session-margin-sm;
-  margin-top: $session-margin-sm;
-}
-
-.session-radio-group fieldset {
-  input[type='radio'] {
-    opacity: 0;
-    position: absolute;
-    cursor: pointer;
-  }
-
-  label {
-    margin-inline-end: 1em;
-    cursor: pointer;
-  }
-
-  label:before {
-    content: '';
-    display: inline-block;
-    width: 0.5em;
-    height: 0.5em;
-    margin-inline-end: 0.8em;
-    border-radius: 100%;
-    vertical-align: 0px;
-    border: 2px solid rgba($session-color-white, 0.6);
-    padding: 0.2em;
-    background: none;
-    border-color: var(--color-accent);
-
-    background-clip: content-box;
-    transition: $session-transition-duration;
-  }
-
-  input:hover + label:before {
-    background: var(--color-accent);
-  }
-
-  input:checked + label:before {
-    background: var(--color-accent);
-  }
-}
-
```

Comment:
Loving all this SASS removal! 😄

Reply:
hope you won't get too many conflicts :x